core-tests/src/test/java/org/mule/runtime/core/DefaultMuleMessageTestCase.java

```diff
@@ -39,7 +40,7 @@
 {
 
     public static final String FOO_PROPERTY = "foo";
-    private Serializable testAttributes = new Apple();
+    private Attributes testAttributes = new Attributes() {};
```

Comment:
This is just a test case, where i want testAttributes to be a unique object instance to then assert with.